server.js

```diff
@@ -1,18 +1,23 @@
 const express = require("express");
 const path = require("path");
+const jsonServer = require("json-server");
+const router = jsonServer.router("db.json");
+const middlewares = jsonServer.defaults();
 
 const app = express();
 const port = process.env.PORT || 3001;
 
+app.use(middlewares);
+app.use("/api", router);
+
```

Comment:
You should move this below the other app.use expressions, so in your case: move it to line - at least that solved the problem for me ;)

Reply:
Thx it works!